packages/ember-engines/addon-test-support/engine-resolver-for.js

```diff
@@ -29,6 +30,19 @@ export default function engineResolverFor(
   modulePrefix = engineName,
   podModulePrefix = undefined
 ) {
+  deprecate(
+    "Use of `engineResolverFor` has been deprecated. Instead use `setupEngine(hooks, 'engine-name')` imported from `ember-engines/test-support` to load the engine you need.",
+    false,
+    {
+      id: 'ember-engines.addon-test-support.engine-resolver-for',
+      for: 'ember-engines',
+      until: '1.0.0',
+      since: {
+        enabled: '0.8.23',
+        available: '0.8.23',
```

Comment:
@villander these actually needs to be `0.8.24` because current version is `0.8.23` and this will be included in the *next* release